client/my-sites/pages/page/index.js

```diff
@@ -73,6 +74,7 @@ class Page extends Component {
 		recordMoreOptions: PropTypes.func.isRequired,
 		recordPageTitle: PropTypes.func.isRequired,
 		recordEditPage: PropTypes.func.isRequired,
+		recordSetFrontPage: PropTypes.func.isRequired,
```

Comment:
The names are different here and below: `recordSetFrontPage` and `recordFrontPage`.

Since this button isn't connected up to the API yet, we probably shouldn't be recording events for it. So we can just remove this line and the other one below.